RTNeural/CMakeLists.txt

```
@@ -46,3 +46,8 @@ target_include_directories(RTNeural
     INTERFACE
         ..
 )
+
+target_compile_definitions(RTNeural
+    PRIVATE
+        RTNEURAL_NAMESPACE=RTNeural
+)
```

```suggestion
set(RTNEURAL_NAMESPACE "RTNeural" CACHE STRING "Namespace to use for RTNeural code")
target_compile_definitions(RTNeural
    PUBLIC
        RTNEURAL_NAMESPACE=${RTNEURAL_NAMESPACE}
)

```
I was thinking it might make sense to do this through a CMake variable, just to be a little more transparent?
I think I'd also prefer the namespace definition to be public so that user-space code can also use `RTNEURAL_NAMESPACE` if needed, but maybe that could cause other issues?